MessageCrackWiz.rc

```diff
@@ -189,6 +189,9 @@ BEGIN
         MENUITEM "Copy &Function\tCtrl+F",      ID_EDIT_COPYFUNCTION
         MENUITEM SEPARATOR
         MENUITEM "Message Fi&lters...\tCtrl+L", ID_EDIT_MESSAGEFILTERS
+        MENUITEM SEPARATOR
+        MENUITEM "Search listbox...\tCtrl+S",   ID_EDIT_SEARCHLISTBOX
+        MENUITEM "Clear All Selects",           ID_EDIT_CLEARALLSELECTS

```

Comment:
proper spelling would be 'Clear selection"  , use a shortcut key with the '&' prefix.